packages/ui-components/package.json

```diff
@@ -43,6 +43,8 @@
 		"clean": "rm -rf node_modules dist loader unit-coverage www docs",
 		"test": "TZ=UTC stencil test --spec --e2e --no-cache",
 		"test:watch": "TZ=UTC stencil test --spec --e2e --watchAll",
+		"test:spec": "TZ=UTC stencil test --spec --no-cache",
```

Comment:
Nice, but please change the `test` script to use these two scripts.

`"test": "npm run test:spec && npm run test:e2e"`